### PR TITLE
Sample groovy script to add credentials to Jenkins

### PR DESCRIPTION
**Pull request by kuisathaverat**
@reviewbybees

### CODE REVIEW

**Comment by imod** [comment]
waiting for @reviewbybees ....

File: scriptler/addCredentials.groovy [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+ ]
+ } END META**/
+
+import com.cloudbees.plugins.credentials.impl.*;
+import com.cloudbees.plugins.credentials.*;
+import com.cloudbees.plugins.credentials.domains.*;
+
+String keyfile = "/tmp/key"
+
+Credentials c = (Credentials) new UsernamePasswordCredentialsImpl(CredentialsScope.GLOBAL,java.util.UUID.randomUUID().toString(), "description", "user", "password")
```
Comment (Evildethow) [inline scriptler/addCredentials.groovy]: Unnecessary qualified reference `java.util.UUID.randomUUID().toString()`

Can just do `UUID.randomUUID().toString()`

Two more occurrences below as well

**Comment by Evildethow** [comment]
🐝

**Comment by Dohbedoh** [comment]
🐝

**Comment by kuisathaverat** [comment]
@reviewbybees done

**Pull request merged**